guarddog/analyzer/metadata/npm/utils.py

```diff
@@ -0,0 +1,5 @@
+def get_email_addresses(package_info: dict) -> list[str]:
+    if package_info.get("maintainers"):
+        return list(map(lambda x: x["email"], package_info["maintainers"]))
```

Comment:
nit: could be a one-liner

```suggestion
        return list(map(lambda x: x["email"], package_info.get("maintainers", [])))
```

Reply:
improved in 289c47e